svm-transaction/src/svm_message.rs

```diff
@@ -104,7 +107,9 @@ pub trait SVMMessage: Debug {
             .and_then(|ix| {
                 ix.accounts.first().and_then(|idx| {
                     let index = usize::from(*idx);
-                    if !self.is_writable(index) {
+                    if (require_static_nonce_account && index >= self.static_account_keys().len())
```

Comment:
It looks like we really only require the number of static account keys; 

I think exposing static account keys is reasonable. Can you think of any reason we'd want to limit the interface to only the number of static account keys?

Reply:
> It looks like we really only require the number of static account keys;

Yeah, just figured we will be using `static_account_keys` more in the future so I exposed the full slice rather than just the length.

> Can you think of any reason we'd want to limit the interface to only the number of static account keys?

Nope, we always have the full list of static account keys available, seems fine to expose it